tests/BasicErasedProvisionTests.fs

```diff
@@ -543,6 +543,42 @@ let ``test basic symbol type ops``() =
    t2T.GetConstructors() |> ignore
    t2T.GetMethod("get_Item1") |> ignore
 
+#if INTERNAL_FSHARP_TYPEPROVIDERS_SDK_TESTS
+
+[<Fact>]
+let ``test reader cache actually caches``() =
+    for i = 1 to 1000 do
```

Comment:
This test looks great.  Do you know the count before the cache is/was fixed?

Reply:
I don't have the actual count as it would be hard to test with a weak reference, but at least the test verifies that we actually do a strong cache.